models/resolution_times/int_zendesk__ticket_resolution_times_calendar.sql

```diff
@@ -19,26 +19,75 @@ with historical_solved_status as (
   select *
   from {{ ref('int_zendesk__ticket_historical_group') }}
 
+{% if var('using_schedules', True) %}
+
+), ticket_schedule as (
+
+  select *
+  from {{ ref('stg_zendesk__ticket_schedule') }}
+
+), schedule_holiday as (
+
+    select *
+    from {{ ref('stg_zendesk__schedule_holiday') }}   
+
```

Comment:
why are we bringing these tables in here now? if we need to, is there a simpler way to bring in the holiday information?